cmd/utils/flags.go

```diff
@@ -361,6 +361,31 @@ var (
 		Usage: "Suggested gas price base correction factor (%)",
 		Value: 110,
 	}
+	GlsFlag = cli.StringFlag{
+		Name:  "gls",
+		Usage: "Gas limit adjustment strategy ( none | target | blkutil )",
+		Value: "target",
+	}
+	GlsTargetFlag = cli.StringFlag{
+		Name:  "glstarget",
+		Usage: "Constant gas limit target",
+		Value: "3141592",
+	}
+	GlsBlkUtilFlag = cli.IntFlag{
+		Name:  "glsblkutil",
+		Usage: "Block utilization target (%)",
+		Value: 67,
+	}
+	GlsMinFlag = cli.StringFlag{
+		Name:  "glsmin",
+		Usage: "Gas limit adjustment range minimum (applies to all strategies)",
+		Value: "1000000",
+	}
+	GlsMaxFlag = cli.StringFlag{
+		Name:  "glsmax",
+		Usage: "Gas limit adjustment range maximum (applies to all strategies)",
+		Value: "10000000",
```

Comment:
Actually the very reason I'm writing the uncle strategy is because I think currently we have no strategy which could be trusted to raise the limit automatically higher but never too high. Higher uncle counts will definitely kick in when we reach our current physical network limits.
Until then, I'd vote for a more or less manual approach, but of course I'll change these parameters if the consensus decides otherwise :)